packages/core/src/components/screens/release-notes/ReleaseNotesScreen.tsx

```diff
@@ -1,6 +1,95 @@
-import React from "react";
-import { Typography } from "@equinor/mad-components";
+import React, { useEffect, useState } from "react";
+import {
+    Accordion,
+    CircularProgress,
+    EDSStyleSheet,
+    Typography,
+    useStyles,
+} from "@equinor/mad-components";
+import { fetchAllReleaseNotes } from "./fetchReleaseNotes";
+import { ChangeLog, Release } from "./ChangeLog";
+import { useEnvironment, useServicePortalName } from "../../../store/mad-config";
+import { getShortDate } from "../../../utils/dateUtils";
+import { ScrollView, View } from "react-native";
+import * as mockData from "../../../static/mock-data/whats-new.json";
+import {useDemoMode} from "../../../store/demo-mode";
 
 export const ReleaseNotesScreen = () => {
-    return <Typography>Not implemented yet</Typography>;
+    const styles = useStyles(releaseNoteStyles);
+    const demoMode = useDemoMode();
+    const environment = useEnvironment();
+    const servicePortalName = useServicePortalName();
+
+    const [releaseNotes, setReleaseNotes] = useState<Release[] | null>(null);
+    const [error, setError] = useState("");
+    const [isFetching, setIsFetching] = useState(true);
+
+    useEffect(() => {
+        if (demoMode.isEnabled) {
+            setReleaseNotes([mockData]);
+            setIsFetching(false);
+        } else {
+            fetchAllReleaseNotes(environment, servicePortalName)
+                .then(setReleaseNotes)
+                .catch((error: Error) => setError(error.message))
+                .finally(() => setIsFetching(false));
+        }
+
+    }, [demoMode.isEnabled, environment, servicePortalName]);
+
+    const sortReleaseNotes = (releaseNoteA: Release, releaseNoteB: Release) => {
+        return releaseNoteA.version < releaseNoteB.version
+            ? releaseNoteB.version < releaseNoteA.version
+                ? -1
+                : 1
+            : 0;
+    };
```

Comment:
This could be it's own hook.
```tsx
export const useReleaseNotes = () => {
    const demoMode = useDemoMode();
    const environment = useEnvironment();
    const servicePortalName = useServicePortalName();

    const [releaseNotes, setReleaseNotes] = useState<Release[] | null>(null);
    const [error, setError] = useState("");
    const [isFetching, setIsFetching] = useState(true);
    // .toSorted returns a sorted array without mutating the original. We don't wanna mutate state
    const sortedReleaseNotes = useMemo(() => releaseNotes && releaseNotes.toSorted(sortReleaseNotes), [releaseNotes])

    useEffect(() => {
        if (demoMode.isEnabled) {
            setReleaseNotes([mockData]);
            setIsFetching(false);
        } else {
            fetchAllReleaseNotes(environment, servicePortalName)
                .then(setReleaseNotes)
                .catch((error: Error) => setError(error.message))
                .finally(() => setIsFetching(false));
        }

    }, [demoMode.isEnabled, environment, servicePortalName]);
    
    return {releaseNotes: sortedReleaseNotes, isFetching, error}
}

const sortReleaseNotes = (releaseNoteA: Release, releaseNoteB: Release) => {
        return releaseNoteA.version < releaseNoteB.version
            ? releaseNoteB.version < releaseNoteA.version
                ? -1
                : 1
            : 0;
    };
```

Reply:
added useReleaseNotes hook